DbcParserLib/Model/Message.cs

```diff
@@ -7,10 +7,10 @@ public class Message
 {
     public uint ID { get; internal set; }
     public bool IsExtID { get; internal set; }
-    public string Name { get; internal set; }
+    public string Name { get; internal set; } = string.Empty;
```

Comment:
this was because i initially set the project to use nullablility which then markes a string as an issue if not defined nullable explicitly or initilaized. 
overall i just dont like Nulls in a public API; expecially in this case here a name is also the key. name would definitly be something to set in the constructor. 
i generally way wondering why mostly no constructors are used. When i would use property initializers instead like that i would still set them to init for set: public string Name {get;init;} which works with the property initializers aswell